sandbox/functions-recipes/includes/durable-activity-functions-errors.md

```diff
@@ -0,0 +1,35 @@
+## Handling activity function exceptions
+Unhandled exceptions thrown in activity functions will be rethrown in the calling orchestrator function as a `FunctionFailedException`. The `InnerException` property of the `FunctionFailedException` will contain the original exception from the activity.
+
+```csharp
+[FunctionName("PlaceOrder")]
+public static async Task OrderOrchestration(
+    [OrchestrationTrigger] DurableOrchestrationContext context,
+    TraceWriter log)
+{
+    try
+    {
+        bool reserved = await context.CallActivityAsync<bool>("CheckAndReserveInventory", null);
+    }
+    catch (FunctionFailedException ex)
+    {
+        log.Error("Inventory check failed", ex);
+    }
+}
+
+
+[FunctionName("CheckAndReserveInventory")]
+public static bool CheckAndReserveInventory([ActivityTrigger] DurableActivityContext context)
+{
+    throw new ArgumentException("Oops...");
+}
+```
+
+[!include[](../includes/takeaways-heading.md)]
+* `FunctionFailedException` is thrown in the orchestrator if an activity function throws an unhandled exception
+* `FunctionFailedException`'s `InnerException` property will contain the source exception from the activity function.
+
+[!include[](../includes/read-more-heading.md)]
+* [Handling errors in Durable Functions](https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-error-handling)
+* [Debugging and Diagnostics of Durable Functions](https://cda.ms/rH)
```

Comment:
What is this URL?  I wouldn't feel safe putting this in our docs given it's not clear who owns the domain (customers probably wouldn't feel safe either).

Also, I wonder if it's better to use the internal *.md links instead of the public HTTP URLs. That's what the other documents use, anyways (though I guess this is probably a different repo).

Reply:
Switching these for expanded links